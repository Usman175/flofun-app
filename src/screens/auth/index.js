import LoginScreen from './LoginScreen'
import SignOut from './SignOut'
import { PhoneSignUpScreen, EnterCodeScreen } from './phone'
import AuthReducer from './reducer'

export { LoginScreen, SignOut, PhoneSignUpScreen, EnterCodeScreen, AuthReducer }
