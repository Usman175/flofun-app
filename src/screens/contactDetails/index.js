import ContactDetailsScreen from './ContactDetailsScreen'
import ContactDetailsReducer from './reducer'

export {
  ContactDetailsScreen,
  ContactDetailsReducer,
}
