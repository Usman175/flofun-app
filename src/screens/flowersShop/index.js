import FlowerShopScreen from './FlowerShopScreen'
import FlowerShopReducer from './reducer'

export {
  FlowerShopScreen,
  FlowerShopReducer,
}
