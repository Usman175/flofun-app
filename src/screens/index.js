export * from './home'
export * from './profile'
export * from './shipping'
export * from './contactDetails'
export * from './payment'
export * from './orderStatus'
