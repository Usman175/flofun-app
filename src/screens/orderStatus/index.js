import OrderStatusScreen from './OrderStatusScreen'

export {
  OrderStatusScreen,
}
