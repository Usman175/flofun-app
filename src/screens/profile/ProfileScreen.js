import React from 'react'
import { Text, View } from 'react-native'
import { Header } from '../../commons'
import styles from './styles/ProfileScreen'

const ProfileScreen = () => (
  <View style={styles.root}>
    <Header title={'Profile'} />
    <View style={styles.content}>
      <Text style={{ color: 'red', backgroundColor: 'blue' }}>Profile page</Text>
    </View>
  </View>
)

export default ProfileScreen
