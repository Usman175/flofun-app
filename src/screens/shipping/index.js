import ShippingScreen from './ShippingScreen'
import ShippingReducer from './reducer'

export {
  ShippingScreen,
  ShippingReducer,
}
