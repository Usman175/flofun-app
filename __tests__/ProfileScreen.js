import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import EStyleSheet from 'react-native-extended-stylesheet'
import { ProfileScreen } from '../src/screens/profile'
import Colors from '../constants/colors'

beforeAll(() => { // eslint-disable-line
  EStyleSheet.build(Colors)
});

describe('ProfileScreen', () => {
  test('renders correctly', () => {
    const profile = renderer.create(
      <ProfileScreen />
    ).toJSON()

    expect(profile).toMatchSnapshot()
  })
})
