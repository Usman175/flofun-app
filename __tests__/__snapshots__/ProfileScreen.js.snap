// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileScreen renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#DDDDDD",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#43405D",
        "flex": 0.1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#fff",
          "fontFamily": "montserratBold",
          "fontSize": 20,
          "textAlign": "center",
        }
      }
    >
      Profile
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 0.9,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "blue",
          "color": "red",
        }
      }
    >
      Profile page
    </Text>
  </View>
</View>
`;
